README.md

```diff
@@ -16,6 +16,8 @@ package in the image.
 
 `openhpc_slurm_service_enabled`: boolean, whether to enable the appropriate slurm service (slurmd/slurmctld)
```

Comment:
Describe optional and defaults (as for other vars)?

Reply:
Thought I'd keep this PR to one logical change.